components/generator/imagevector/src/test/kotlin/io/github/composegears/valkyrie/generator/imagevector/lazyproperty/SvgGradientParserTest.kt

```diff
@@ -0,0 +1,370 @@
+package io.github.composegears.valkyrie.generator.imagevector.lazyproperty
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import io.github.composegears.valkyrie.generator.imagevector.ImageVectorGenerator
+import io.github.composegears.valkyrie.generator.imagevector.ImageVectorGeneratorConfig
+import io.github.composegears.valkyrie.generator.imagevector.OutputFormat.LazyDelegateProperty
+import io.github.composegears.valkyrie.generator.imagevector.backingproperty.loadIcon
+import io.github.composegears.valkyrie.parser.IconParser
+import org.junit.jupiter.api.Test
+
+class SvgGradientParserTest {
+
+    @Test
+    fun `svg linear gradient parsing`() {
+        val icon = loadIcon("ic_linear_gradient.svg")
+        val parserOutput = IconParser.toVector(icon)
+        val output = ImageVectorGenerator.convert(
+            vector = parserOutput.vector,
+            kotlinName = parserOutput.kotlinName,
+            config = ImageVectorGeneratorConfig(
+                packageName = "io.github.composegears.valkyrie.icons",
+                packName = "",
+                nestedPackName = "",
+                outputFormat = LazyDelegateProperty,
+                generatePreview = false,
+            ),
+        ).content
+
+        val expectedOutput = """
+            package io.github.composegears.valkyrie.icons
+
+            import androidx.compose.ui.geometry.Offset
+            import androidx.compose.ui.graphics.Brush
+            import androidx.compose.ui.graphics.Color
+            import androidx.compose.ui.graphics.SolidColor
+            import androidx.compose.ui.graphics.vector.ImageVector
+            import androidx.compose.ui.graphics.vector.path
+            import androidx.compose.ui.unit.dp
+            import kotlin.LazyThreadSafetyMode
+
+            val LinearGradient: ImageVector by lazy(LazyThreadSafetyMode.NONE) {
```

Comment:
These text fixtures could be written to `resources/*.kt` and read, for shortening tests.

Reply:
Tests now very short ❤️